paasta_tools/cli/cmds/logs.py

```diff
@@ -121,18 +152,31 @@ def completer_clusters(prefix, parsed_args, **kwargs):
         return list_clusters()
 
 
+def color_component(component, color_param):
+    # If the color field is iterable try to join
+    # all the colors together, otherwise just apply
+    # the single color
+    try:
+        for color in color_param:
```

Comment:
Can you explain the use case for this? If I'm reading this correctly, this will go through a list of functions, applying each of them to the same argument? Is this so you can have something like ``strip(blue("foo"))``?